docs/source/alignment.rst

```diff
@@ -0,0 +1,68 @@
+**********************
+Verification Alignment
+**********************
+
+A forecast is verified by comparing a set of initializations at a given lead to
+observations over some window of time. However, there are a few ways to decide *which*
+initializations or verification window to use.
+
+One can pass the keyword ``alignment=...`` to any compute functions (e.g.,
+:py:func:`~climpred.prediction.compute_persistence`,
+:py:func:`~climpred.prediction.compute_hindcast`,
+:py:meth:`~climpred.classes.HindcastEnsemble.verify`) to change the behavior for
+aligning forecasts with the verification product.
+
+The available keywords are:
+
+*  ``'same_init'`` (current default): Use a common set of initializations that verify
```

Comment:
have we decided whether we change this? 
for bias and s2s we should: Hawkins, Ed, Buwen Dong, Jon Robson, Rowan Sutton, and Doug Smith. “The Interpretation and Use of Biases in Decadal Climate Predictions.” Journal of Climate 27, no. 8 (January 13, 2014): 2931–47. https://doi.org/10/f5w2xr.

not 100% sure for DCPP: Boer, G. J., D. M. Smith, C. Cassou, F. Doblas-Reyes, G. Danabasoglu, B. Kirtman, Y. Kushnir, et al. “The Decadal Climate Prediction Project (DCPP) Contribution to CMIP6.” Geosci. Model Dev. 9, no. 10 (October 25, 2016): 3751–77. https://doi.org/10/f89qdf.

Reply:
This is just the default since it's the only mode available with this PR. I think we should change to `same_verif` as the standard once it's implemented?